parser/prism/Translator.cc

```diff
@@ -4130,7 +4130,14 @@ Translator::translateParametersNode(pm_parameters_node *paramsNode, core::LocOff
         params.emplace_back(translate(paramsNode->rest));
     }
 
-    translateMultiInto(params, posts);
+    for (auto &prismNode : posts) {
+        // We can skip NoKeywordsParameterNodes in the posts list, they are only valid in `keyword_rest`. This only
+        // happens for invalid syntax like `def foo(**nil, **nil); end`.
```

Comment:
I suggest elaborating this more, and why we only check `posts` and not the others (otherwise it just looks like we didn't consider it)

```suggestion
        // Valid Ruby can only have `**nil` once in the parameter list, which is modelled with a
        // `NoKeywordsParameterNode` in the `keyword_rest` field.
        // If invalid code tries to use more than one `**nil` (like `def foo(**nil, **nil)`),
        // Prism will report an error, but still place the excess `**nil` nodes in `posts` list (never the others like
        // `requireds` or `optionals`), which we need to skip here.
```